ding2.install

```diff
@@ -576,9 +576,7 @@ function ding2_update_7047() {
  * Enable ding_app_variables module.
  */
 function ding2_update_7048() {
-  if (module_exists('opensearch')) {
-    module_enable(array('ding_app_variables'));
-  }
+  module_enable(array('ding_app_variables'));
```

Comment:
This will enable `ding_app_variables` and it's dependencies on all sites - even if they do not have Opensearch.

Sites decided whether to enable `opensearch` or not in `ting_update_7014()` which was a part of the system since 7.x-4.3.0. If we can risk running the update hook on earlier sites, then we should declare the dependency on that hook using `hook_update_dependencies()`.

Reply:
I took the liberty of adding a change for this in 33ec233.